models/oneinch/fantom/oneinch_fantom_calls_transfers.sql

```diff
@@ -0,0 +1,127 @@
+{{ config( 
+    schema = 'oneinch_fantom',
+    alias = alias('calls_transfers'),
+    tags = ['dunesql'],
+    partition_by = ['_partition'],
```

Comment:
we're looking to keep a universal name for this column, so if you don't mind going back to `block_month`

Reply:
while you may not use the column, some spells might and it'll be better to query on the app